_benchmark/metrics/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Metrics
+nav_order: 25
+has_children: true
+---
+
+# Metrics
+
+After a workload completes, OpenSearch Benchmark stores all metric records within its metrics store. These metrics can be kept in memory or inside of an OpenSearch cluster. 
+
+## Storing metrics
+
+You set where metrics are stored using the [`datastore.type`](https://opensearch.org/docs/latest/benchmark/configuring-benchmark/#results_publishing) parameter in your `benchmark.ini` file. 
+
+### In memory
+
+If you want to store your result in memory, use the following settings in the `results_publishing` section of `benchmark.ini`:
+
+```json
+[results_publishing]
+datastore.type = in-memory
+datastore.host = 
+datastore.port = 
+datastore.secure = False
+datastore.ssl.verification_mode = 
+datastore.user = 
+datastore.password =
+```
+
+### OpenSearch
+
+If you want to store your results in an external OpenSearch memory store, use the following settings in the `results_publishing` section of `benchmark.ini`:
+
+```
```

Comment:
`datastore.number_of_replicas` and `datastore.number_of_shards` settings are missing. Please add them. Also need to mentioned that if not provided then the defaults provided by OpenSearch are used which is 0 for replica and 1 for shard, please confirm. 
Also if the setting is applied after OpenSearch datastore cluster has been creaetd and results have been published, the replica and shards setting will only apply when new results indices are created at the start of each month.